pkg/api/common.go

```diff
@@ -145,6 +145,11 @@ func (r *RedirectResponse) WriteTo(ctx *models.ReqContext) {
 	ctx.Redirect(r.location)
 }
 
+// Status gets the response's status.
+func (*RedirectResponse) Status() int {
```

Comment:
Will this method ever be called? Just wondering, since any build failure can also be solved by adding `Response` on line 141.

Reply:
It seems that this function was added in the interface only for testing purposes and is only used in the `librarypanels` service so no it won't be called rn. But it could be useful for future tests. 

I don't see how adding `Response` on line 141 will avoid us to add this function.

Reply:
@AgnesToulet If you add `Response` on line 141, it will make `RedirectResponse` have default implementations of all the `Response` methods. The only technical difference would be that calling `Status()` on a `RedirectResponse` would fail. I've used this technique to implement large interfaces, where you don't want to write all the methods.